main/at-spi2-atk/APKBUILD

```diff
@@ -1,45 +1,37 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
-# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=at-spi2-atk
-pkgver=2.26.2
-pkgrel=1
+pkgver=2.32.0
+pkgrel=0
 pkgdesc="A GTK+ module that bridges ATK to D-Bus at-spi"
 url="https://www.freedesktop.org/wiki/Accessibility/AT-SPI2/"
 arch="all"
 options="!check"  # Requires dbus daemon running.
 license="LGPL-2.0-or-later"
-depends=""
-depends_dev="dbus-dev atk-dev"
```

Comment:
Why should `-dev` no longer have these dependencies?

Reply:
abuild warned me that these are already automagically added. Will add a notice to the commit msg.

Reply:
Thank you